doc/guide.md

```diff
@@ -3188,29 +3196,51 @@ etc.  also load sets of reference data. This can be controlled by
 supplying tenant parameters. These are properties (key-value pairs)
 that are passed to the module when enabled or upgraded. Passing those
 are only performed when tenantParameters is specified for install and
-when the tenant interface is version 1.2.
+when the tenant interface is version 1.2 and later.
 
 In FOLIO two such parameters are widely recognized:
 
  * `loadReference` with value `true` loads reference data.
  * `loadSample` with value `true` loads sample data.
 
-### Tenant Interface
+#### Tenant Interface definitions
 
-The full `_tenant` interface version 1.1/1.2 portion:
+A module supporting 1.1/1.2 of the `_tenant` interface should use this
+snippet in the module descriptor:
 
 ```
    "id" : "_tenant",
    "version" : "1.2",
    "interfaceType" : "system",
    "handlers" : [ {
-     "methods" : [ "POST", "DELETE" ],
-     "pathPattern" : "/_/tenant"
-    }, {
-     "methods" : [ "POST" ],
-     "pathPattern" : "/_/tenant/disable"
-    } ]
+       "methods" : [ "POST", "DELETE" ],
+       "pathPattern" : "/_/tenant"
+      }, {
+       "methods" : [ "POST" ],
+       "pathPattern" : "/_/tenant/disable"
+      }
+   ]
+```
+The corresponding RAML definition:
+[tenant.raml](https://github.com/folio-org/raml/blob/tenant_interface_1_2/ramls/tenant.raml)
+
+Snippet of version 2.0 of the `_tenant` interface:
+
 ```
+   "id" : "_tenant",
+   "version" : "2.0",
+   "interfaceType" : "system",
+   "handlers" : [ {
+       "methods" : [ "POST" ],
+       "pathPattern" : "/_/tenant"
+      }, {
+       "methods" : [ "GET", "DELETE" ],
+       "pathPattern" : "/_/tenant/{id}"
+      }
+   ]
```

Comment:
Show we add `"permissionsRequired" : [ ]`?

Reply:
We could, but kind of redundant. They aren't required for system interfaces.

Reply:
Agreed.

Reply:
I thought it was agreed that all endpoints in module descriptors *should* declare an empty `modulePermissions` property to make it explicit that a conscious decision had been made.

@craigmcnally @skoczko Did I misunderstand that guidance?